ci/deploy.sh

```diff
@@ -1,50 +1,26 @@
 #!/bin/bash
 
-# called by Travis CI
-
 set -ex
 
-if [[ "false" != "$TRAVIS_PULL_REQUEST" ]]; then
-	echo "Not deploying pull requests."
-	exit
-fi
-
-if [[ "$TRAVIS_BRANCH" != "$DEPLOY_BRANCH" ]]; then
-	echo "Not on the '${DEPLOY_BRANCH}' branch."
-	exit
-fi
+# Convert the URLs in the superproject .gitmodules file,
+# then init those submodules
+sed -i -e "s|git@\([^:]*\):|https://\1/|" .gitmodules
+git submodule update --init
+# Now recurse over all the contained submodules,
+# sub-submodules, etc, to do the same
+date; git submodule foreach --recursive 'if [ -w .gitmodules ]; then sed -i -e "s|git@\([^:]*\):|https://\1/|" "$toplevel/$path/.gitmodules"; fi; git submodule update --init "$toplevel/$path";'; date;
 
 DEPLOY_BUILD_DIR="/tmp/deploy_build/"
 
-# Anyone can read the build log, so it MUST NOT contain any sensitive data,
-# turn off script echoing for a bit
-set +x
-
-# Nuke the existing SSH key
-rm -fv ~/.ssh/id_rsa
-
-# The private portion of the deploy key is in a Travis repository
-# setting called VIP_GITHUB_BUILD_REPO_DEPLOY_KEY. Before pasting
-# it into the setting, I replaced newlines with \n and surrounded
-# it with double quotes, e.g. "KEY\nHERE\n".
-echo -e $VIP_GITHUB_BUILD_REPO_DEPLOY_KEY > ~/.ssh/id_rsa
-chmod 600 ~/.ssh/id_rsa
+cp ./ci/known_hosts ~/.ssh/known_hosts
 
-# Restore script echoing now we've done the private things
-set -x
-
-cp ${TRAVIS_BUILD_DIR}/ci/known_hosts ~/.ssh/known_hosts
-
-git clone git@github.com:Automattic/vip-go-mu-plugins-built.git /tmp/target
+git clone  git@github.com:Automattic/vip-go-mu-plugins-built.git /tmp/target
```

Comment:
Super small nit-pick: there's a space added between `clone` and `git@` that's not necessary.

Reply:
Fixed.